components/generic/RelatedCollections.vue

```diff
@@ -29,50 +30,59 @@
     },
 
     props: {
-      query: {
+      title: {
```

Comment:
Is this prop still necessary if the title is "Collections you might like" everywhere?

Reply:
I can remove it but it was added when we had multiple titles and if in future multiple titles are requested again then they can be added easily.